node_test/rpc_test.go

```diff
@@ -78,10 +86,11 @@ func executeNRpcTest(t *testing.T, connectionType transport.TransportType, n int
 	// Setup
 	//////////////////////
 
-	t.Logf("Starting test with %d clients", n)
 	logFile := fmt.Sprintf("test_%d_rpc_clients_over_%s.log", n, connectionType)
 	logDestination := newLogWriter(logFile)
 	defer logDestination.Close()
+	logger := testLogger(logDestination)
```

Comment:
I don't have super strong feelings about this, but here the engines and the test output write to the same log file, which reproduces the maybe-too-noisy outputs currently being ignored by readers of failed test runs.

Existing `t.Logf` calls should not be contributors to any logging performance issues, since they print to stdout only if the test fails.

Reply:
That's a good point. As an fyi, we run unit tests [with -v flag in CI](https://github.com/statechannels/go-nitro/blob/e7d8a0585f363be1a1fa2730b3186ab81bd75ba1/.github/workflows/go.yml#L34). So if we leave `logf` in the codebase, we should turn off the verbose flag.

I also don't feel strongly about this change. The benefit of the proposed change is that all logs are in a single file. The added log lines will not grow the log file significantly.